atomic_reactor/buildimage.py

```diff
@@ -112,6 +113,7 @@ def get_reactor_tarball_path(self, tmpdir):
         os.chdir(local_reactor_git_path)
         try:
             logger.debug("executing sdist command in directory '%s'", os.getcwd())
+            sys.stderr.flush()
```

Comment:
Rather than flushing sys.stderr after every logging call, wouldn't it be more robust to modify EncodedStream (from __init__.py) so it calls `self.binarystream.flush()` after writing?